### ISSUE TITLE
what are input_arrays and output_arrays for the model

### ISSUE THREAD

**Issue by samaa** [deepmind/kinetics-i3d#69]
hi. iam trying to export the model into tensorflow lite but i don't know what are the input_arrays and output arrays required for the conversion method
converter = tf.lite.TFLiteConverter.from_frozen_graph(
  graph_def_file, input_arrays, output_arrays)
 one  more issue how can i save the model from a module as tflite. I am getting a graph error all the time
 m = hub.Module("https://tfhub.dev/deepmind/i3d-kinetics-400/1")
  INPUT_SIZE = (None, 224, 224, 3)
  model = tf.keras.Sequential([
    hub.KerasLayer(m,output_shape=[400], dtype=tf.float32, trainable=False), 
    tf.keras.layers.Dropout(rate=0.2)
  ])
